src/vaults/Vault.sol

```diff
@@ -721,15 +718,13 @@ abstract contract Vault is IVault, CoreReference, ReentrancyGuardUpgradeable, Va
     /// @notice sets a new value for the token0 floor
     /// @param _token0FloorNum the new floor token0 returns (out of `DENOM`)
     function setToken0Floor(uint256 _token0FloorNum) external override onlyStrategist {
-        require(_token0FloorNum > 0, "INVALID_TOKEN0_FLOOR");
```

Comment:
Note the `param` natspec comment – this is out of `DENOM`. We won't be setting the floor to 0 (that means the users would be receiving 0% back!), we'll be setting it to 1 (or, `10_000`) which is the value of `DENOM`